watchfiles/main.py

```diff
@@ -319,3 +331,10 @@ def _auto_force_polling() -> bool:
 
     uname = platform.uname()
     return 'microsoft-standard' in uname.release.lower() and uname.system.lower() == 'linux'
+
+
+def _default_ignore_permission_denied(ignore_permission_denied: Optional[bool]) -> bool:
+    if ignore_permission_denied is not None:
+        return ignore_permission_denied
+    env_var = os.getenv('WATCHFILES_IGNORE_PERMISSION_DENIED')
+    return env_var is not None
```

Comment:
```suggestion
    return bool(env_var)
```
To follow the convention that empty strings are equivalent to unsetting the env var.